bedevere/bpo.py

```diff
@@ -111,6 +110,25 @@ def create_success_status(found_issue):
                               target_url=url)
 
 
+def create_failure_status_issue_not_on_bpo(issue_number_found):
+    """Create a failure status for when an issue does not exist on the bug tracker."""
+    issue_number = issue_number_found.group("issue")
```

Comment:
What is the benefit of not passing in the issue number directly instead of the `re.Match` object?

Reply:
See above.